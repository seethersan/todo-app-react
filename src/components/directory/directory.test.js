import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Directory from './directory.component';
import '@testing-library/jest-dom';

describe('expect to render Directory component', () => {
    var mockState= {
        tasks: [],
        currentView: 'done'
    }

    it('expect to render Directory component', () => {        
        expect(shallow(<Directory {...mockState} />)).toMatchSnapshot();
    });

    it('expect to render done', () => {
        const wrapper = shallow(<Directory {...mockState} />);
        expect(wrapper.find("h2.title").text().trim()).toBe('DONE Items');
    });

    it('expect to render todo', () => {
        mockState.currentView = 'todo';
        const wrapper = shallow(<Directory {...mockState} />);
        expect(wrapper.find("h2.title").text().trim()).toBe('TODO Items');
    });
})

