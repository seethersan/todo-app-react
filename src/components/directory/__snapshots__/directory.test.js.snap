// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Directory component expect to render Directory component 1`] = `ShallowWrapper {}`;
